### ISSUE TITLE
ADD loss in epoch 120

### ISSUE THREAD

**Issue by AB1675** [GANWANSHUI/ES6D#17]
Thank you for your interesting work. 
Could you please tell me how much was the ADD loss in epoch 120 (latest epoch) in training data? log file is not in your GitHub.
I would be grateful if you let me know as soon as possible. 
Thanks

**Comment by GANWANSHUI** [comment]
We did not retrain the model with ADD(S) loss. If you mean the A(M)GPD loss, what I could find in the training info about the proposed A(M)GPD loss (GADD) at 120 epoch is _**E120 GADD_loss:0.0226 GADD:0.0789**_ . Another way is to reload the released trained model and test it to see the loss.

**Comment by AB1675** [comment]
Thank you so much. Yeah, I mean GADD loss.
Thanks!
